src/pages/dataset-details-page/index.tsx

```diff
@@ -177,7 +177,7 @@ const DatasetDetailsPage: FC<Props> = ({
           ?.flatMap(({ accessService }) => accessService?.map(({ uri }) => uri))
           ?.filter((accessUri): accessUri is string => !!accessUri) ?? [];
       if (accessUris.length > 0) {
-        getDataServices({ uris: accessUris });
```

Comment:
legg til støtte for size, med verdi som er lengden til accessUris, i tilfelle det er flere enn 10